.github/workflows/linter.yml

```diff
@@ -40,8 +40,6 @@ jobs:
           #   - .*\.(in|SKELETON) - all template/skeleton files
           #   - tools/coverity\.sh - external file (with some modifications)
```

Comment:
@mrc0mmand I'm not sure `tools/coverity.sh` should be ignored. It's kind of convoluted though. I'd just replace it with something like https://github.com/util-linux/util-linux/commit/c881bec5fe10eff9cf06e2db4dc479d798f9eae6#diff-b08ad0c56cb48fcac947f04aa72b28c7f500e1accf1c827831e693ecee0474fc and unblock superlinter. I'm not sure why I haven't done it here yet :-)

Reply:
I tried to change it and broke something. Probably it is wise to ignore it as long as it works :-)